backend/config/config.go

```diff
@@ -638,6 +646,17 @@ func (c *Config) PostProcess() error {
 
 }
 
+func (c *Config) arrangeSmtpSettings() {
+	if c.Passcode.Smtp.Validate() == nil {
+		if c.Smtp.Validate() == nil {
+			zeroLogger.Warn().Msg("Both smtp and passcode.smtp are set. Using smtp settings from passcode.smtp")
```

Comment:
This warning message is wrong, the root smtp settings will be used.

Reply:
Ah, thank you. Updated.